velox/exec/Aggregate.h

```diff
@@ -65,6 +65,15 @@ class Aggregate {
     return true;
   }
 
+  // Returns the size for a single-value variable length
+  // accumulator. For example, map_agg has 4xthe initial array size as
+  // overhead even for a single key-value pair. This is on top of
+  // accumulatorFixedWidthBytes() and the serialized size of the
```

Comment:
typo: accumulatorFixedWidthBytes -> accumulatorFixedWidthSize

The description of the method is not very clear. What is "the size for a single-value variable length accumulator". Is this an estimate of the size of the variable-width part of the accumulator? Are there any suggestions for function authors on how to calculate these estimates?

It would be more robust for default implementation to throw if isFixedSize is false. Otherwise, it is easy to forget to implement this method.